buildlib/tools/test_mad.sh

```diff
@@ -45,7 +47,13 @@ docker_run_srv() {
         -v /hpc/local:/hpc/local \
         --ulimit memlock=-1:-1 --device=/dev/infiniband/ \
         $IMAGE \
-        bash -c "${PWD}/install/bin/ucx_perftest -K ${HCA}"
+        bash -c "${PWD}/install/bin/ucx_perftest -K ${HCA} > \
+            ${PWD}/ucx_perf_srv_${test_name}_${BUILD_BUILDID}.log 2>&1"
+}
+
+srv_logs() {
```

Comment:
maybe we can print the logs during server stop stage?

Reply:
Good idea. The only drawback is that you will need to know where to look for logs, rather than dedicated steps now.

Reply:
Add ".. and print logs" to the stage name :)